content/applications/websites/website/configuration/translate.rst

```diff
@@ -2,101 +2,111 @@
 
 .. tip::
-   - Once the languages have been installed, you can also translate some items (e.g.,
-     the product's name and description) from the backend (e.g., in the product template). To do so,
-     click the language code (e.g., :guilabel:`EN`) next to the text you want to translate (e.g.,
-     the product name) and add the translation.
+  You can also add the :guilabel:`Language selector` in the :guilabel:`Header` of your page. To do
+  so, click the :guilabel:`Header` block and go to the :guilabel:`Navbar` section to edit the
+  :guilabel:`Language selector`.
 
-     .. image:: translate/product-translation.png
-        :alt: Translate product-related items.
+.. _translate/translate:
 
-   - You can also :doc:`export/import translations <../../../../developer/howtos/translations>`
-     to translate multiple items (e.g., product names and descriptions) in one go.
+Translate your website
+======================
 
-.. _translate/language-selector:
+Select your desired language from the :guilabel:`Language selector` to see your content in another
+language. Then, click the :guilabel:`Translate` button in the top-right corner to manually activate
+the translation mode so that you can translate what has not been translated automatically by Odoo.
 
-Language selector menu
-======================
+Translated text strings are highlighted in green; Text strings that were not translated
+automatically are highlighted in yellow.
 
-To add a language selector menu:
+.. image:: translate/translated-text.png
+  :alt: Entering the translation mode
 
-#. Go to your website and click :guilabel:`Edit`.
-#. Select the block where you want to add the language selector menu (e.g., the header).
-#. Select the :guilabel:`Customize` tab.
-#. In the :guilabel:`Navbar` section, set the :guilabel:`Language selector` field to either
-   :guilabel:`Dropdown` or :guilabel:`Inline`.
+In this mode, you can only translate texts. To change the page's structure, you must edit the master
+page. Each modification on the master page is automatically applied to all translated versions.
+
+.. note::
+  When a website supports multiple languages, the content is accessible through different URLs,
+  depending on the language selected.
+  `https://www.mywebsite.com/shop/product/my-product-1` --> URL to the master page, in English.
+  `https://www.mywebsite.com/fr_FR/shop/product/mon-produit-1` --> URL to the French version.
 
-   .. image:: translate/language-selector.png
-      :alt: Add a language selector menu.
+To replace the original text with the translation, click the block, edit its contents, and save.
 
-#. Click :guilabel:`Save`.
+.. tip::
+  Once the desired language is installed, you can translate some items from the backend (e.g., the
+  :guilabel:`product’s name` in the product template). To do so, click :guilabel:`EN` next to the
+  text you want to translate and add the translation.
+
+Block customization per language
+--------------------------------
```

Comment:
- I would not call this section "Block customization" since you can't really customize the block based on the language, you can just hide/show it (like you explained directly below).
- Also, from 17.0, you could just refer to the "Conditional visibility" section in the Building blocks doc and then just keep the last bullet point in your list if you want ("Go to languages to configure ..." - you'll probably need to change it a little bit so it fits the text).